specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json

```diff
@@ -18,10 +18,11 @@
     }
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+    "hostTemplate": "{Endpoint}",
+    "useSchemePrefix": false,
     "parameters": [
       {
-        "$ref": "#/parameters/AzureRegion"
+        "$ref": "#/parameters/Endpoint"
```

Comment:
Endpoint has to be "$ref", otherwise, it behaves like  "x-ms-parameter-location": "method".

Reply:
I believe you'll have to update all the examples, (like [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/examples/SuccessfulAnalyzeWithStream.json#L3)).

Reply:
@cthrash Thanks, didn't know about the examples, they are now updated.